tools/build/cross-build/include/linux/string.h

```diff
@@ -49,8 +49,8 @@
 #include <sys/cdefs.h>
 
 __BEGIN_DECLS
-size_t strlcpy(char *dst, const char *src, size_t siz);
-size_t strlcat(char *dst, const char *src, size_t siz);
+size_t strlcpy(char * __restrict dst, const char * __restrict src, size_t siz);
+size_t strlcat(char * __restrict dst, const char * __restrict src, size_t siz);
```

Comment:
These are prototypes for glibc's functions (or another Linux libc), not FreeBSD's, used when building native tools at the start of the build.